test/Removal.t.sol

```diff
@@ -837,3 +861,42 @@ contract Removal_safeBatchTransferFrom_reverts_ForbiddenTransfer is
     });
   }
 }
+
```

Comment:
for these just wanted to experiment to see that these functions are pausable by way of the pausable overrides of their internal implementations inside `PausableAccessPreset`.  As reading in the solidity docs, I believe that (for example and WLOG) since `grantRole` calls `_grantRole` (and not `super._grantRole`), it will resolve `_grantRole` starting with the most derived contract possible and walking up the inheritance tree from there, so indeed it calls the version defined in `PausableAccessPreset` and not the version defined in its own class